webui/react/src/pages/Dashboard.tsx

```diff
@@ -57,6 +59,18 @@ const Dashboard: React.FC = () => {
   });
   const [ filters, setFilters ] = useState<TaskFilters>(initFilters);
 
+  const fetchExperiments = useCallback(async (): Promise<void> => {
+    const response = await getExperimentList(
+      { descend: true, key: 'startTime' },
```

Comment:
an alternative here could be to pass `sortEnum.startTime`

Reply:
but then I assume the usecase with antd table would be more difficult to map the key to enum?